diff --git a/cloudify_kubernetes/__version__.py b/cloudify_kubernetes/__version__.py
index 7345107..ef58aee 100644
--- a/cloudify_kubernetes/__version__.py
+++ b/cloudify_kubernetes/__version__.py
@@ -1 +1 @@
-version = '2.13.27'
+version = '2.13.28'
diff --git a/constraints.txt b/constraints.txt
index 863be00..f63d8c6 100644
--- a/constraints.txt
+++ b/constraints.txt
@@ -1,3 +1 @@
-cloudify-common>===6.4.2
-git+https://github.com/cloudify-cosmo/cloudify-manager.git@6.4.2-build#egg=cloudify-types&subdirectory=cloudify_types
 charset_normalizer===3.0.1
\ No newline at end of file
diff --git a/dev-requirements.txt b/dev-requirements.txt
index 19732bb..e69de29 100644
--- a/dev-requirements.txt
+++ b/dev-requirements.txt
@@ -1 +0,0 @@
-git+https://github.com/cloudify-cosmo/cloudify-manager.git@6.4.2-build#egg=cloudify-types&subdirectory=cloudify_types
diff --git a/examples/blueprint-examples b/examples/blueprint-examples
--- a/examples/blueprint-examples
+++ b/examples/blueprint-examples
@@ -1 +1 @@
-Subproject commit 706cac79e74217733c4c4cf26032021ba55132f8
+Subproject commit 706cac79e74217733c4c4cf26032021ba55132f8-dirty
diff --git a/setup.py b/setup.py
index f4d4d02..64097ce 100755
--- a/setup.py
+++ b/setup.py
@@ -31,10 +31,9 @@ def get_version():
 
 install_requires = [
     'cloudify-python-importer==0.2.1',
-    'cloudify-common>=4.5',
-    'cloudify-types>=6.3.1',
-    'deepdiff==3.3.0',
-    'cloudify-utilities-plugins-sdk>=0.0.112',  # Provides kubernetes and google-auth.
+    'fusion-common',
+    'deepdiff==5.7.0',
+    'cloudify-utilities-plugins-sdk',  # Provides kubernetes and google-auth.
 ]
 
 
diff --git a/test-requirements.txt b/test-requirements.txt
index 2904754..4253e91 100644
--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -4,4 +4,4 @@ mock
 nose-testconfig
 nose==1.3.7
 nose-cov==1.6
-deepdiff==3.3.0
+deepdiff==5.7.0
