diff --git a/constraints.txt b/constraints.txt
index 863be00..26c55dc 100644
--- a/constraints.txt
+++ b/constraints.txt
@@ -1,3 +1,2 @@
-cloudify-common>===6.4.2
-git+https://github.com/cloudify-cosmo/cloudify-manager.git@6.4.2-build#egg=cloudify-types&subdirectory=cloudify_types
-charset_normalizer===3.0.1
\ No newline at end of file
+cryptography>=41.0.4
+charset_normalizer===3.0.1
diff --git a/dev-requirements.txt b/dev-requirements.txt
index 19732bb..1ce86c2 100644
--- a/dev-requirements.txt
+++ b/dev-requirements.txt
@@ -1 +1,4 @@
-git+https://github.com/cloudify-cosmo/cloudify-manager.git@6.4.2-build#egg=cloudify-types&subdirectory=cloudify_types
+-e fusion-common
+-e fusion-agent
+-e fusion-manager/mgmtworker
+-e cloudify-utilities-plugins-sdk
diff --git a/examples/blueprint-examples b/examples/blueprint-examples
--- a/examples/blueprint-examples
+++ b/examples/blueprint-examples
@@ -1 +1 @@
-Subproject commit 706cac79e74217733c4c4cf26032021ba55132f8
+Subproject commit 706cac79e74217733c4c4cf26032021ba55132f8-dirty
diff --git a/setup.py b/setup.py
index f4d4d02..0e54075 100755
--- a/setup.py
+++ b/setup.py
@@ -16,7 +16,7 @@ import os
 import re
 import sys
 import pathlib
-from setuptools import setup
+from setuptools import setup, find_packages
 
 PY2 = sys.version_info[0] == 2
 
@@ -31,10 +31,10 @@ def get_version():
 
 install_requires = [
     'cloudify-python-importer==0.2.1',
-    'cloudify-common>=4.5',
-    'cloudify-types>=6.3.1',
-    'deepdiff==3.3.0',
-    'cloudify-utilities-plugins-sdk>=0.0.112',  # Provides kubernetes and google-auth.
+    'fusion-common',
+    'fusion-mgmtworker',
+    'deepdiff==5.7.0',
+    'cloudify-utilities-plugins-sdk>=0.0.129',  # Provides kubernetes and google-auth.
 ]
 
 
@@ -45,10 +45,7 @@ setup(
     author_email='hello@cloudify.co',
     description='Plugin provides Kubernetes management possibility',
     include_package_data=True,
-    packages=['cloudify_kubernetes',
-              'cloudify_kubernetes.k8s',
-              'cloudify_kubernetes.tasks',
-              'cloudify_kubernetes.tasks.nested_resources'],
+    packages=find_packages(),
     license='LICENSE',
     install_requires=install_requires
 )
